docs/guide/packages/lucide-react.md

```diff
@@ -91,7 +91,7 @@ export default Icon;
 import Icon from './Icon';
 
 const App = () => {
-  return <Icon name="Home" />;
+  return <Icon name="home" />;
```

Comment:
The icons object is pointing to this index files. Which includes the names of the modules in pascal case.
<img width="458" alt="image" src="https://github.com/lucide-icons/lucide/assets/11825403/74f53a54-8e9b-48b1-b209-82fa75f29350">

Reply:
Oh, sorry. I was using it in dymanic import. I thought they were using the same case. I'm wrong.